tests/driver_bme280_saul/main.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2016 Kees Bakker, SODAQ
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the BME280 temperature, pressure
+ *              and humidity sensor.
+ *
+ * @author      Kees Bakker <kees@sodaq.com>
+ *
+ * @}
+ */
+
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "bme280_params.h"
```

Comment:
Those 2 includes (bme280*) are not needed. Maybe you have to include "saul.h". To be tested.